src/win32.js

```diff
@@ -98,15 +99,53 @@ class WindowsApp extends App {
     }
   }
 
+  async signAppIfSpecified () {
+    const windowsSignOpt = this.opts.windowsSign
+    const windowsMetaData = this.opts.win32metadata
+
+    if (windowsSignOpt) {
+      const signOpts = createSignOpts(windowsSignOpt, windowsMetaData, this.renamedAppPath)
+      debug(`Running @electron/windows-sign with the options ${JSON.stringify(signOpts)}`)
+      try {
+        await sign(signOpts)
+      } catch (err) {
+        // Although not signed successfully, the application is packed.
+        if (signOpts.continueOnError) {
+          common.warning(`Code sign failed; please retry manually. ${err}`, this.opts.quiet)
+        } else {
+          throw err
+        }
+      }
+    }
+  }
+
   async create () {
     await this.initialize()
     await this.renameElectron()
     await this.copyExtraResources()
     await this.runRcedit()
+    await this.signAppIfSpecified()
```

Comment:
Is this definitely run after the `afterCopy` hook? Just double checking this is run _after_ any potential mutations to the exe.  E.g. via fuses

Reply:
Yep! The only thing that happens after is moving the package to its final destination and the `afterComplete` hook. For future convenience, this is also exactly when we sign & notarize on macOS.